test/SequenceEntity/SequenceEntity.js

```diff
@@ -59,6 +59,23 @@ describe('SequenceEntity', () => {
 			}, 50);
 		});
 
+		it('Check completion', done => {
+			const sirenSequenceRoot =  window.D2L.Hypermedia.Siren.Parse(sequenceRootMultipleTopLevel);
+			const completionData = {};
+			entityFactory(SequenceEntity, '/sequenceRoot2', 'whatever', (entity) => {
+				entity.onSubSequencesChange((subSequence) => {
+					completionData[subSequence.index()] = subSequence.completion();
+				});
+			}, sirenSequenceRoot);
+
+			setTimeout(() => {
```

Comment:
Why do we use `setTimeout`? Isn't this going to be flaky?

Reply:
Jeff has a fix but right now it is the only way I could do it.